src/includes/constants.js

```diff
@@ -295,4 +295,4 @@ pub.DEFAULTMODE = pub.MODEVISIBLE; // FIXME, DEFAULTMODE shouldn't be public, mo
 
 
 var ERROR_PREFIX = "\nBasil.js Error -> ",
-    WARNING_PREFIX = "### Basil Warning -> ";
\ No newline at end of file
+  WARNING_PREFIX = "### Basil Warning -> ";
```

Comment:
Hm, is there any chance we can leave this sort of formatting? I think it looks just cleaner, if it is aligned. In this specific case it's not such a drama, but what about cases like [this one](https://github.com/basiljs/basil.js/blob/master/src/includes/environment.js#L657). Would it not be possible to keep those?

Just my personal opinion, if others disagree, go ahead and lint it like this.